src/components/number-picker/number-picker.component.ts

```diff
@@ -154,8 +163,13 @@ export class NumberPickerComponent implements ControlValueAccessor {
         if (value !== undefined) {
             this._value = value;
             this._valid = this.isValid();
+
+            // if the component is not destroyed then run change detection
+            // workaround for Angular bug (https://portal.digitalsafe.net/browse/EL-3694)
+            if (!this._isDestroyed) {
```

Comment:
Could we fix the formatting here? Consider adding this to your VS Code settings as this will automatically format your TypeScript code correctly on save:

```json
"[typescript]": {
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.organizeImports": true
        },
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
```

Reply:
No worries, I will do it now

Reply:
Thanks